week-9/Homework/mandatory/2-exercises/exercise.js

```diff
@@ -0,0 +1,33 @@
+/*
+
+Complete the rest of this code to create an online shopping cart.
+
+The output of running your code should be:
+
+    Your shopping cart has 3 items: toilet roll, pasta, eggs
+*/
+
+class ShoppingCart {
+  // Add your code here
+
+  constructor(){
+    this.cart='Your shopping cart has 3 items: '
```

Comment:
wouldn't it be better to count how many items are there in the cart, instead of hardcoding 3 items?